docs/adding_tracks.rst

```diff
@@ -530,26 +527,26 @@ Finally, specify the compressed file name in your ``track.json`` file in the ``s
 
 Specifying ``compressed-bytes`` (file size of ``documents.json.bz2``) and ``uncompressed-bytes`` (file size of ``documents.json``) is optional but helps Rally to provide progress indicators and also verify integrity.
 
-You've now mastered the basics of track development for Rally. It's time to pat yourself on the back before you dive into the advanced topics!
+You've now mastered the basics of track development for Rally. It's time to pat yourself on the back before you dive into the advanced topics.
 
 Advanced topics
 ---------------
 
 Template Language
 ^^^^^^^^^^^^^^^^^
 
-Rally uses `Jinja2 <http://jinja.pocoo.org/docs/dev/>`_ as template language. This allows you to use Jinja2 expressions in track files.
+Rally uses `Jinja2 <http://jinja.pocoo.org/docs/dev/>`_ as template language so you can use Jinja2 expressions in track files.
 
 
 Extension Points
 """"""""""""""""
 
 Rally also provides a few extension points to Jinja2:
```

Comment:
How about just `a few extensions` (without points)?

Reply:
That's simpler indeed. I also changed the heading accordingly.